File: tests/dynamic_fixtures/java/sqli_unsupported.java
// SQL injection — unsupported fixture.
// Entry is an instance method rather than a static method.
// Test sets confidence = Low to get Unsupported(ConfidenceTooLow).
// Expected verdict: Unsupported

public class Entry {
    public void findUser(String name) {
        String query = "SELECT * FROM users WHERE name='" + name + "'";
        System.out.println(query);
    }
}

File: tests/dynamic_fixtures/go/sqli_unsupported.go
// SQL injection — unsupported fixture.
// Entry is a method on a struct — entry kind unsupported (only Function supported).
// Test sets confidence = Low to get Unsupported(ConfidenceTooLow).
// Entry: UserRepo.FindUser  Cap: SQL_QUERY
// Expected verdict: Unsupported

package entry

import "fmt"

type UserRepo struct{}

func (r *UserRepo) FindUser(name string) {
	query := "SELECT * FROM users WHERE name='" + name + "'"
	fmt.Println(query)
}
